app/pages/nfmCapturePhonePage.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages
+
+import play.api.libs.json.JsPath
+
+case object nfmCapturePhonePage extends QuestionPage[String] {
```

Comment:
All case objects should have upper camel case
```suggestion
case object NFMCapturePhonePage extends QuestionPage[String] {
```
or
```suggestion
case object NfmCapturePhonePage extends QuestionPage[String] {
```

Reply:
Done